kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -469,6 +472,111 @@ class SchemaUtilsSuite extends AnyFunSuite {
       }.asJava)
   }
 
+  ///////////////////////////////////////////////////////////////////////////
+  // validateUpdatedSchema
```

Comment:
These are all pretty simple tests for SchemaUtils, again I think the real investment in testing should be at the API layer. So we'll be porting many of the tests https://github.com/delta-io/delta/pull/4196/files to API tests. It doesn't provide much value to essentially duplicate all of those at the SchemaUtils level.